build/SkiaSharp.props

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup>
-    <PackageReference Include="SkiaSharp" Version="2.88.6" />
+    <PackageReference Include="SkiaSharp" Version="2.88.8-preview.1.1" />
```

Comment:
Right now the NuGet is just a preview, but once I manage to get all the other things ready and it lives for a few days and gets a few download I will push it stable.